### PR TITLE
Clean up a few (friend) declarations of test functions in CpGrid.hpp

### PR DESCRIPTION
**Pull request by aritorto**
This PR removes a few (friend) declarations of free test functions, in CpGrid.hpp. 
Not relevant for the Reference Manual.

### CODE REVIEW

**Comment by aritorto** [comment]
jenkins build this please

**Comment by aritorto** [comment]
jenkins build this please

File: opm/grid/cpgrid/CpGridData.hpp [hunk 1]
```diff
@@ -182,15 +180,12 @@ class CpGridData
...
-    friend
-    void ::lookup_check(const Dune::CpGrid&);
     friend
     void ::fieldProp_check(const Dune::CpGrid& grid, Opm::EclipseGrid eclGrid, std::string deck_string);
     friend
     void ::testInactiveCellsLgrs(const std::string&,
-                           const std::vector<std::array<int,3>>&,
-                           const std::vector<std::array<int,3>>&,
+                                 const std::vector<std::array<int,3>>&,
+                                 const std::vector<std::array<int,3>>&,
```
Comment (blattms) [inline opm/grid/cpgrid/CpGridData.hpp:188]: Is indentation correct, below here?

**Review by blattms — COMMENTED** [review]
Nice, Just one question/nitpick.

File: opm/grid/cpgrid/CpGridData.hpp (hunk 1 above)
Comment (aritorto) [reply opm/grid/cpgrid/CpGridData.hpp:188]: Ops, I must have one that without intention. I'll correct indentation

**Comment by aritorto** [comment]
jenkins build this please

**Pull request merged**